internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -40,6 +42,7 @@ var (
 	errInvalidSidecarDependsOnStatus = fmt.Errorf("sidecar container dependency status must be one of < %s | %s | %s >", dependsOnStart, dependsOnComplete, dependsOnSuccess)
 	errEssentialContainerStatus      = fmt.Errorf("essential container dependencies can only have status < %s | %s >", dependsOnStart, dependsOnHealthy)
 	errEssentialSidecarStatus        = fmt.Errorf("essential sidecar container dependencies can only have status < %s >", dependsOnStart)
+	errInvalidPubSubName             = errors.New("topic/worker names can only contain letters, numbers, underscores, and hypthens")
```

Comment:
```suggestion
	errInvalidPublishField             = errors.New("topic/worker names can only contain letters, numbers, underscores, and hypthens")
```

Reply:
I wonder if it's worthy to have different err message for topic and workers, so that people know exactly which of topic or worker has invalid name?

Reply:
Separated them out with some new naming validation suggested above!